LayoutTests/imported/w3c/web-platform-tests/css/css-grid/alignment/grid-align-baseline-fieldset-001-expected.txt

```diff
@@ -14,6 +14,12 @@ FAIL .target > * 2 assert_equals:
 offsetTop expected 0 but got 10
 FAIL .target > * 3 assert_equals:
 <div data-offset-y="40"><span></span></div>
-offsetTop expected 40 but got 20
-PASS .target > * 4
```

Comment:
PASS -> FAIL. Is this work tracked by one of the other last-baseline patches?

Reply:
Keeping track of this issue here: https://bugs.webkit.org/show_bug.cgi?id=246413